genisys/modules/preseed.py

```diff
@@ -4,51 +4,58 @@
 from genisys.modules.base import Module
 from genisys.config_parser import YAMLParser
 import sys
-FILENAME = "preseed.cfg"
 
+FILENAME = "preseed.cfg"
+CERTIFICATE_STORE_PATH = Path("/etc/genisys/ssl")
 class Preseed(Module):
     """Generates a Preseed file to be served over the network to a booting Debian system"""
     def __init__(self: Self, config: YAMLParser):
         self.config = {}
         self.config["network"] = config.get_section("Network")
         self.config["users"] = config.get_section("Users")
         self.config["apps"] = config.get_section("Applications")
-    # end __init__
 
     def install_location(self: Self) -> Path:
         """Places the Preseed file in the TFTP root"""
         ftp_dir = Path(self.config["network"]["tftp_directory"])
         return Path(ftp_dir, FILENAME)
-    # end install_location
 
     def generate(self: Self) -> str:
         """Renders the Jinja template with the prespecified configurations"""
         loader = jinja2.PackageLoader('genisys')
         jinja_env = jinja2.Environment(loader=loader)
         template = jinja_env.get_template("preseed.cfg.jinja2")
 
-        # convert booleans to lowercase to match preseed format
+        # Convert booleans to lowercase to match preseed format
         for key, value in self.config["users"].items():
             if isinstance(value, bool):
                 self.config["users"][key] = str(value).lower()
 
+                # Read SSH key files and store their contents as strings
+                ssh_keys_contents = []
+                ssh_keys_dir = Path(self.config["users"].get("ssh-keys-dir", "../ssh_keys"))
```

Comment:
"ssh-keys-dir" isn't documented in the example config. I was under the impression that the "ssh-keys" section would all be absolute paths to the relevant keys (so this wouldn't be needed). Either way is fine, we just need one way and it needs to be properly documented in the example.yaml.

Reply:
I have changed the code to only have absolute paths for ssh and also documented it in the example.yml